vmdk_plugin/plugin.go

```diff
@@ -117,19 +116,53 @@ func (d *vmdkDriver) mountVolume(name string) (string, error) {
 		return mountpoint, err
 	}
 
+	if d.useMockEsx {
+		return mountpoint, fmt.Errorf("No device to mount.")
+	}
+
+	watcher, err := inotify.NewWatcher()
+	if err != nil {
+		log.WithFields(
+			log.Fields{"name": name, "dir": mountpoint},
+		).Error("Failed to create watcher ")
+		return mountpoint, err
+	}
+	err = watcher.Watch("/dev/disk/by-path")
```

Comment:
Is it possible to watch the specific device (do the watch on the device thats to be mounted. Do it on the device returned by get_device_path(). Else will the watcher return if some device gets created, not necessarily the one for the attach below.